modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -22,6 +22,10 @@ assetPublisherDisplayContext.setPageKeywords();
 if (assetPublisherDisplayContext.isEnableTagBasedNavigation() && !assetPublisherDisplayContext.isSelectionStyleAssetList() && assetPublisherDisplayContext.isSelectionStyleManual() && (ArrayUtil.isNotEmpty(assetPublisherDisplayContext.getAllAssetCategoryIds()) || ArrayUtil.isNotEmpty(assetPublisherDisplayContext.getAllAssetTagNames()))) {
 	assetPublisherDisplayContext.setSelectionStyle(AssetPublisherSelectionStyleConstants.TYPE_DYNAMIC);
 }
+
+List<Group> selectedGroups = GroupLocalServiceUtil.getGroups(assetPublisherDisplayContext.getGroupIds());
+
+request.setAttribute("selectedGroups", selectedGroups);
```

Comment:
Is there a reason we fetch the groups here, only to have `categorization_filter/page.jsp` use the groupIds?

Reply:
Hi @holatuwol, 
With others file the **groupIds** will be got from **assetPublisherDisplayContext** that is initialized in **init.jsp** file. But this **categorization_filter/page.jsp** file is in **asset-taglib** module so it it can't import **AssetPublisherDisplayContext** from **asset-publisher-web** to get **groupIds**. So I decide to set it in **view.jsp**.
Beside, it should just need a list of **groupIds** instead of **groups**. So I will change it a little bit and resend it.
Is it ok with you?

Reply:
I updated the fix. Please help me review it. Thank you.